### PR TITLE
Create hanzas.thedev.me.json

### PR DESCRIPTION
**Pull request by hanzas09**

### CODE REVIEW

**Review by YazeedT — CHANGES_REQUESTED** [review]
Remove the trailing dot in your MX records and all should be good to merge.

File: domains/hanzas.thedev.me.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        ],
+        "CNAME": [
+            "hanzas09.github.io"
+        ],
+        "NS": [
+            "ns1.example.com",
+            "ns2.example.com"
+        ],
+        "MX": [
+            "mx1.improvmx.com.",
```
Comment (YazeedT) [inline domains/hanzas.thedev.me.json:28]: Remove the trailing dot in your MX records and all should be good to merge.

File: domains/hanzas.thedev.me.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            "2606:50c0:8002::153",
+            "2606:50c0:8003::153"
+        ],
+        "CNAME": [
+            "hanzas09.github.io"
+        ],
+        "NS": [
+            "ns1.example.com",
+            "ns2.example.com"
+        ],
```
Comment (YazeedT) [inline domains/hanzas.thedev.me.json]: Remove the trailing dot in your MX records and all should be good to merge.

Comment (hanzas09) [reply domains/hanzas.thedev.me.json]: Ok Done.

**Review by hanzas09 — COMMENTED** [review]
Fixed the NS, MX Records

**Review by hanzas09 — COMMENTED** [review]
Here we go again. Removed extra records

**Review by hanzas09 — COMMENTED** [review]
Copied a config from a old user and modified

**Pull request merged**